app/controllers/concerns/ip_geo_country_helper.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module IpGeoCountryHelper
+  def set_geo_country
+    db = MaxMindDB.new("./GeoLite2-Country.mmdb")
+    db.lookup(request.remote_ip).country.iso_code
```

Comment:
In Load Balancer, this will return private IP, not original request IP.
Check X-Forwarded-For, X-Forwarded-Proto, and X-Forwarded-Port headers that DO load balancer sets to pass information about original request. And if they're empty, we'll need to change load balancer configuration.

Reply:
https://docs.digitalocean.com/products/kubernetes/how-to/configure-load-balancers/#preserving-client-source-ip-address